### PR TITLE
(refactor) some opportunities to use lambda functions

### PR DESCRIPTION
**Pull request by desmorto**

### CODE REVIEW

**Comment by desmorto** [comment]
Sorry, i don't understand how would be the correct project identation for these chained lambdas

**Comment by iluwatar** [comment]
@desmorto the build is failing because of Checkstyle issue. Can you check out this guide to configure Checkstyle for your IDE and correct the errors?

https://github.com/iluwatar/java-design-patterns/wiki/12.-IDE-instructions

File: hexagonal/src/main/java/com/iluwatar/hexagonal/database/MongoTicketRepository.java [hunk 1]
```diff
@@ -169,10 +169,9 @@ public MongoDatabase getMongoDatabase() {
   public Map<LotteryTicketId, LotteryTicket> findAll() {
     Map<LotteryTicketId, LotteryTicket> map = new HashMap<>();
     List<Document> docs = ticketsCollection.find(new Document()).into(new ArrayList<Document>());
-    for (Document doc: docs) {
-      LotteryTicket lotteryTicket = docToTicket(doc);
+    docs.stream().map(lotteryTicket -> docToTicket(doc)).forEach(lotteryTicket -> {
```
Comment (rastdeepanshu) [inline hexagonal/src/main/java/com/iluwatar/hexagonal/database/MongoTicketRepository.java]: Can you break it into multiple lines for easy readability? Like:
docs.stream()
    .map(...)
    .forEach(...)

**Comment by piotrek1543** [comment]
I am not owner or collaborator of this repo, but I give my approve as I haven't found anything to improve or fix.

**Comment by iluwatar** [comment]
Thanks for reviewing @rastdeepanshu and @piotrek1543 I agree it looks good but the build is failing because of Checkstyle issues and those should be fixed before the pull request can be merged.

**Comment by iluwatar** [comment]
Pull request closed due to inactivity

**Pull request closed without merging**